terraform/panther_log_analysis_iam/main.tf

```diff
@@ -45,7 +45,10 @@ resource "aws_iam_role_policy" "log_processing" {
     Statement : [
       {
         Effect : "Allow",
-        Action : "s3:GetBucketLocation",
+        Action : [
+          "s3:GetBucketLocation",
+          "s3:ListBucket"
+        ],
```

Comment:
Also the `GetObject` has to change. See https://github.com/panther-labs/panther-auxiliary/blob/main/cloudformation/panther-log-analysis-iam.yml#L117

@giorgosp it seems that the Terraform template doesn't include the changes that were made in CFN. Can you help @jzandona with this?

Reply:
@jzandona could we make this change too?

```
diff --git a/terraform/panther_log_analysis_iam/main.tf b/terraform/panther_log_analysis_iam/main.tf
index 0e8e518..a06020c 100644
--- a/terraform/panther_log_analysis_iam/main.tf
+++ b/terraform/panther_log_analysis_iam/main.tf
@@ -54,7 +54,7 @@ resource "aws_iam_role_policy" "log_processing" {
       {
         Effect : "Allow",
         Action : "s3:GetObject",
-        Resource : "arn:aws:s3:::${var.s3_bucket_name}/${var.s3_prefix}*"
+        Resource : "arn:aws:s3:::${var.s3_bucket_name}/*"
       },
       {
         Effect : "Allow",
```

I don't have write access to this repo to commit it myself

Reply:
Thanks for catching this, updated.